butterfly-cli/src/main/java/com/paypal/butterfly/cli/ButterflyCliRunner.java

```diff
@@ -132,7 +145,7 @@ public ButterflyCliRun run() throws IOException {
             logger.info("-z option has been set, transformed application will be placed into a zip file");
         }
 
-        Configuration configuration = new Configuration(transformedApplicationFolder, createZip);
+        Configuration configuration = new Configuration(transformedApplicationFolder, createZip, optionSet.has(CLI_OPTION_IN_PLACE_TRANSFORM));
```

Comment:
If `-f` is specified, then you can ignore the `transformedApplicationFolder` and `createZip`, and just use `Configuration` default constructor instead.